.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+# Config for https://github.com/release-drafter/release-drafter
+name-template: 'Akka Serverless Java SDK $NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Productboard Features'
```

Comment:
Just `Features` (without `Productboard`) for SDK repos?

Reply:
I agree, just Features is better since this is open source